apps/api/src/app/workflows/e2e/update-notification-template.e2e.ts

```diff
@@ -33,7 +33,13 @@ describe('Update workflow by id - /workflows/:workflowId (PUT)', async () => {
             {
               template: {
                 type: StepTypeEnum.IN_APP,
-                content: 'This is new content for notification variant',
+                content: 'first content',
+              },
+            },
+            {
+              template: {
+                type: StepTypeEnum.IN_APP,
+                content: 'second content',
```

Comment:
@djabarovgeorge just a note - it should not be possible to create/update a variant with no conditions

Reply:
Updates or creates. Variants can not have empty conditions